ui/src/alerting/actions/notifications/rules.ts

```diff
@@ -67,9 +68,8 @@ export const getNotificationRules = () => async (
       throw new Error(resp.data.message)
     }
 
-    dispatch(
-      setAllNotificationRules(RemoteDataState.Done, resp.data.notificationRules)
-    )
+    // TODO(watts): replace with resp.data.notificationRules when implemented
+    dispatch(setAllNotificationRules(RemoteDataState.Done, [mockRule]))
```

Comment:
Not relevant here but just on my mind: whenever we have a `DraftSomethingOrOther` resource, we'll need to convert to the draft form whenever receiving from the server, and convert back before persisting changes. I wonder if there's a certain approach we should be taking for this generally applicable problem.

In Ember for example, this is the responsibility of a ["serializer"](https://guides.emberjs.com/release/models/customizing-serializers/).